docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Importer direct upload
```

Comment:
Suggest rewording to something like "Improved Hue Importer -- Select a file, choose a dialect, create a table".